src/SDKs/SqlManagement/Sql.Tests/ManagedDatabaseVulnerabilityAssessmentTestFixture.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.Azure.Management.ResourceManager.Models;
+using Microsoft.Azure.Management.Sql;
+using Microsoft.Azure.Management.Sql.Models;
+using Sku = Microsoft.Azure.Management.Sql.Models.Sku;
+
+namespace Sql.Tests
+{
+    public class ManagedDatabaseVulnerabilityAssessmentTestFixture : IDisposable
+    {
+        public SqlManagementTestContext Context { get; set; }
+
+        public ResourceGroup ResourceGroup { get; set; }
+
+        public ManagedInstance ManagedInstance { get; set; }
+
+        public ManagedDatabaseVulnerabilityAssessmentTestFixture()
+        {
+            Context = new SqlManagementTestContext(this);
+
+            try
+            {
+                SqlManagementClient sqlClient = Context.GetClient<SqlManagementClient>();
+
+                ResourceGroup = Context.CreateResourceGroup();
+
+                Sku sku = new Sku();
+                sku.Name = "MIGP8G4";
+                sku.Tier = "GeneralPurpose";
+                ManagedInstance = sqlClient.ManagedInstances.CreateOrUpdate(ResourceGroup.Name,
+                    "crud-tests-" + SqlManagementTestUtilities.GenerateName(), new ManagedInstance()
+                    {
+                        AdministratorLogin = SqlManagementTestUtilities.DefaultLogin,
+                        AdministratorLoginPassword = SqlManagementTestUtilities.DefaultPassword,
+                        Sku = sku,
+                        SubnetId =
+                            "/subscriptions/a8c9a924-06c0-4bde-9788-e7b1370969e1/resourceGroups/RG_MIPlayground/providers/Microsoft.Network/virtualNetworks/VNET_MIPlayground/subnets/MISubnet",
```

Comment:
This subnet id shouldn't be hardcoded. Tests must not require any precreated dependencies. Please create the subnet in test code.

Reply:
After consulting with the managed instance team I created the managed instance the same way that it is created here - https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/SqlManagement/Sql.Tests/ManagedInstanceCrudScenarioTests.cs